### DISCUSSION TITLE
How to use SubNav

### DISCUSSION THREAD

**Discussion by pchaganti** [primer/react#3054] [Q&A] [answered]
Using Primer for a project. Everything is working very well so far. Thanks for a well thought out design system!

One issue we are facing is trying to get the SubNav component to work. Anyone have a small working sample for SubNav?

thanks for any pointers!

**Comment by lesliecdubs** [comment, marked as answer]
Hi @pchaganti, thanks for the kind words! Can you share any more information about where you are running into problems with getting `SubNav` to work? Are you able to provide an example or recreation in a gist or Codepen for a Primer maintainer to review?

**Reply by pchaganti** [reply]
Was able to finally get it to work by starting from a clean project!

thanks!

**Reply by lesliecdubs** [reply]
Great news 🎉 Thanks for letting us know!
